frontend/tests/acceptance/gitlab-test.js

```diff
@@ -14,10 +14,6 @@ import moment from 'moment'
 module('Acceptance | GitLab', function(hooks) {
   setupApplicationTest(hooks)
 
-  hooks.beforeEach(function() {
-    this.application = null
-  })
-
   hooks.beforeEach(async function() {
     this.application = startApp()
```

Comment:
https://github.com/emberjs/ember-qunit/blob/master/addon-test-support/ember-qunit/index.js#L142 `startApp` should not be necessary!